src/_guides/language/numbers.md

```diff
@@ -0,0 +1,547 @@
+in a web program. It might be easier for me to explain if I could see
+examples of the underlying types you're talking about.]***
+
+For native targets, you can assume that
+`int` maps to a signed 64-bit integer representation and
+`double` maps to a 64-bit, IEEE floating-point representation
+that matches the underlying processor:
+
+```
+  Object
+     |
+    num
+   /    \
+ int   double 
+  |       |
+native* native*
+  int   double
+```
+***[TODO: Make this a real figure]***
+
+In general, you can ignore the platform-specific types,
+and think of `int` and `double` as concrete implementation types.
+
+{{ site.alert.info }}
+  **Implementation note:**
+  Dart represents `int` and `double` in
+  a few different ways for efficiency,
+  but these are hidden (marked `*` above) and
+  beyond the scope of this discussion.
+  ***[QUESTION: Can we explain this differently?
+  Drop this note?
+  I don't understand what the \* really means.
+  An example might help.]***
+{{ site.alert.end }}
+
+The web, where Dart compiles to and interoperates with JavaScript,
+has a single numeric representation:
+a 64-bit, double-precision floating point value.
+For efficiency, Dart maps both `int` and `double` to this single representation.
+The visible type hierarchy remains the same,
+but the underlying hidden implementation types (marked `*`) are
+different and intertwined:
+
+```
+   Object
+     |
+    num
+   /    \
+ int   double 
+  |   /   |
+  |  /    |
+  js*     js*
+  int   double
+```
+***[TODO: Make this a real figure]***
+
+Specifically, an `int` is represented as
+a double-precision floating point value with no fractional part.
+In practice, this works pretty well.
+Double-precision floating point provides 53 bits of integer precision.
+However, it means that `int` values are always also `double` values,
+which can lead to some surprises.
+
+
+## Differences in behavior
+
+Most integer and double arithmetic
+has essentially the same behavior.
+There are, however, important differences —
+particularly when your code has strict expectations about
+precision, string formatting, or underlying runtime types.
+
+When arithmetic results differ, as described in this section,
+the behavior is **platform specific**
+and **subject to change**.
+
+{{ site.alert.note }}
+  The platform-specific behavior might change to be
+  less surprising, more consistent, or more performant.
+{{ site.alert.end }}
+
+
+### Precision
+
+The following table demonstrates how some numerical expressions
+differ due to precision.
+Here, `math` represents the `dart:math` library,
+and `math.pow(2, 53)` is 2<sup>53</sup>.
+
+On the web, integers lose precision past 53 bits.
+In particular, 2<sup>53</sup> and 2<sup>53</sup>+1
+map to the same value due to truncation.
+***[QUESTION: The language tour's
+[numbers section](https://dart.dev/guides/language/language-tour#numbers)
+says 2<sup>53</sup> - 1, not 2<sup>53</sup>. Should I update it?]***
```

Comment:
I *think* these statements are consistent.  I.e., we cannot tell the difference between 2^53 and 2^53+1.

Reply:
But we can tell the difference between 2^53 - 1 and 2^53...